src/auth_api/app.py

```diff
@@ -26,6 +26,8 @@
     AcceptTerms,
     # User:
     GetUserInformation,
+    # User Uuid:
+    GetUserUuid,
```

Comment:
Generally look at the changes you are looking to merge in, and change `user` to `company`. This includes the endpoint path ;)

Reply:
I agree